src/documents/WebGamesAndSimulations.md

```diff
@@ -0,0 +1,164 @@
+
+**Scope of this Guide:** This guide deals with interactive games built using standard web technologies such as HTML5, CSS, SVG, and JavaScript. By using established web standards, the content you create has a higher chance of being understood by assistive technologies.
+
+## Orienting the User
+
+Regardless of the modality used for interaction, a logical, predictable structure helps users focus on learning and the experience instead of getting stuck on the interaction. Consistency and predicability is important.
+
+Help orient the user by:
+
+* having a consistent structure.
+* describe the scene and setting so non-visual users can understand their context.
+* identifying the actors and interactive parts.
+* giving meaningful and predictable landmarks for the user.
+* use progress and status indicators as necessary.
+* try to keep the structure stable. Be mindful about removing items completely from the structure - consider disabling elements rather than removing them.
+
+## Player Choices and Controls
+
+Some people make choices easily, while others require more time before making decisions. Choices in games can be exciting for some, but stressful for others.
+
+If presenting learners with multiple choices that affect the outcome of a game:
+
+* If possible, make choices forgiving: give opportunity for users to recover from or change their choices.
+* Avoid player choices which require decisions within a time limit or provide options to disable such features.
+
+Other best practices are:
+* Label each button and control. Labels should use simple terminology and define any terms if necessary.
+* Labels should state simply what the button will do. If longer descriptions are needed, use an `aria-describedby` property.
+* Buttons, links, controls should make it clear what the outcome would be. For example: A choice called "I'm out of here" could be changed to "I'm out of here - quit game", or simply "Quit".
+* In some cases text where text labels are not visible (example: icon buttons that do not have visible text labels), consider using this technique for making invisible content for screen readers: <a href="http://webaim.org/techniques/css/invisiblecontent/" rel="nofollow" target="_blank" class="link-external">"Invisible Content Just for Screen Reader Users" on WebAim.org</a>.
+
+## Communicating Events and Actions with Text and Sound
+
+A large part of interactive content are the events that happen during and after a user performs an action. Often these occur as on-screen visuals, animations, and transitions. The following are some techniques for creating an equivalent experience for non-sighted learners:
+
+* Use ARIA live-regions to communicate changes to the interface.
+* Use ARIA `describedby` to write text descriptions of what has happened.
```

Comment:
These mentions of ARIA techniques might benefit from links to relevant articles explaining the techniques?